sql/core/src/main/scala/org/apache/spark/sql/execution/datasources/FileSourceStrategy.scala

```diff
@@ -192,10 +192,7 @@ object FileSourceStrategy extends Strategy with PredicateHelper with Logging {
           Some(f)
         }
       }
-      val supportNestedPredicatePushdown =
-        DataSourceUtils.supportNestedPredicatePushdown(fsRelation)
-      val pushedFilters = dataFilters
-        .flatMap(DataSourceStrategy.translateFilter(_, supportNestedPredicatePushdown))
+      val pushedFilters = DataSourceStrategy.pushedDownFilters(dataFilters, fsRelation)
       logInfo(s"Pushed Filters: ${pushedFilters.mkString(",")}")
```

Comment:
This is only used for logging. Can we move the logging into `FileSourceScanExec`?

Reply:
[DataSourceStrategy.scala#L701-L706](https://github.com/apache/spark/blob/ac4f2af8d9184db3962427133ef2940c8b515c26/sql/core/src/main/scala/org/apache/spark/sql/execution/datasources/DataSourceStrategy.scala#L701-L706) also needs to ensure filter is pushable.